api/segmentwriter/v1/push.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+
+package segmentwriter.v1;
+
+import "types/v1/types.proto";
+
+service SegmentWriterService {
+  rpc Push(PushRequest) returns (PushResponse) {}
+}
+
+message PushResponse {}
+
+// WriteRawRequest writes a pprof profile
+message PushRequest {
+  // series is a set raw pprof profiles and accompanying labels
+  repeated RawProfileSeries series = 1;
+}
```

Comment:
We discussed this internally at some point, and I recall the consensus was that batching does not benefit us here. On the contrary, it introduces several issues:
1. Callers have to wait all the affected shard segment writers to flush, which badly impacts latency, and may also impact resource usage on the distributor side.
2. It complicates error handling. I'm not 100% sure that partial success is handled properly.  
3. It complicates retries on the distributor end.

I hope we'll amend the API and implementation accordingly in follow-up PRs.

Reply:
I removed repeated series, but kept repeated samples